core/tag_api.php

```diff
@@ -529,6 +529,25 @@ function tag_get_field( $p_tag_id, $p_field_name ) {
 	}
 }
 
+/**
+ * Can the specified user create a tag?
+ *
+ * @param integer $p_user_id     The user ID to link the new tag to.
```

Comment:
Wrong comment `The user ID to link the new tag to`

Reply:
Fixed.